Makefile

```diff
@@ -24,7 +24,7 @@ test: test-ios
 
 	
 test-ios:
-	xcodebuild -workspace Kinvey.xcworkspace -scheme Kinvey -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.2' -enableCodeCoverage YES test
+	xcodebuild -workspace Kinvey.xcworkspace -scheme Kinvey -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.1' -enableCodeCoverage YES test
```

Comment:
this is to fix a problem in Travis CI... don't know why, but Travis CI has 2 iOS Simulator with 10.2 and for that reason the script ask you to specify the UUID of the Simulator (which can also change from time to time)